chutney/server/src/main/java/com/chutneytesting/execution/infra/aop/IndexingAspect.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.execution.infra.aop;
+
+import com.chutneytesting.execution.infra.storage.jpa.ScenarioExecutionReportEntity;
+import com.chutneytesting.index.infra.ScenarioExecutionReportIndexRepository;
+import java.util.List;
+import java.util.Set;
+import org.aspectj.lang.annotation.After;
+import org.aspectj.lang.annotation.Aspect;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+public class IndexingAspect {
```

Comment:
I don't know if it's a good idea to index a running scenario report.
I would plugin the indexing of report only when it has a final status.